### DISCUSSION TITLE
Lower heroic key requirements early due to long RDF waits.

### DISCUSSION THREAD

**Discussion by Zelfie** [chromiecraft/chromiecraft#5877] [Suggestions]
Alright so hear me out on this one.
The population is quite low as it is, and some reputations, especially Keepers of Time, require a lot more dungeon grinding that can't be min-maxed with quests.  Even with min-maxing the most forgiving reps, it still requires a good normal dungeon grind to get revered and unlock heroic keys.
Now that most players are farming heroics, the amount of normal dungeon groups via RDF has shrunk massively. And if mythic dungeons are enabled, even less of the population are going to want to do normals, and you're making the barrier for entry into this content (that you're pushing so much) harder by how long RDF queues will be for normal dungeons. I've sat in normal RDF queues for hours in the evening and just can't imagine farming heroic keys on alts, especially when I'm not playing a tank/healer.

You can get your guild to help!
Yeah, I can, but that's not really solving the problem, that's just putting the weight and responsibility on their shoulders to carry me or others. What about the quiet solo players or small guilds? More players getting into heroics, and potentially mythics, means more heroic groups, more content being played.
I see long normal RDF wait times potentially being harmful to the community for anyone who hasn't got the rep to push into heroic content. If I was 1x dungeon away from heroic key, i'd wait an hour and beg for help. But if I needed 6x more, oof that's a big ask to sit around and wait every evening.

**Comment by KostovDimitar** [comment]
Keys at Revered require a huge population, indeed. And if the Mythic rewards are the HC Epics without enchants or improvements this will stimulate people to play and want to gear more alts which will contribute big time to the improvement of Normal queues.

**Comment by Peti321** [comment]
Most of the people just wait around for the raids to get better gear from. The players willing to do RDF will only decline. It's bearable in Classic dungeons, you can ask guildies / world chat for some runs, however in TBC the dungeons are essetial parts of progression. Blizzard did lower the requirements even in on the main servers. It's okay to lower it sooner then them.

**Comment by Egor-aux** [comment]
I agree, I don't think that the current rep progression system fits the player base, adjusting it would be a great move imo.

**Comment by Nachovski** [comment]
With raids coming, rep requierement probably should be downgraded to Honorable. Newcomers (and alts) wont be able to join the begining of the Tbc raiding experience that many are waiting for. Raids for sure will increase a bit the population. We want to receive them, not enslave them to grind

**Comment by Amrx2** [comment]
Hi all, it's already planned for keys to drop to honored with the release of raids (which will be in July should all go well).

**Reply by Zelfie** [reply]
Yeah I know, we're asking if it could be brought forward a little. Give newbs and alts a chance to get prepared and join those opening raids, hit the ground running, as opposed to waiting in the sidelines for heroics when everyone has moved onto raids.
Say 2 weeks, open the rep up 2 weeks before raids, that's 2 weeks to push the much smaller rep grind, and plenty of heroic lockout for even the busiest players. Newbs and alts have plenty of chance to join in the first week of raiding without getting left behind, and heroic dungeons get a surge in numbers before raids start popping, instead of competing against guilds pushing their first raids.

**Reply by Amrx2** [reply]
That makes more sense, thanks. Lets see what we can come up with.

**Comment by balleny** [comment]
To put in some arguments against an earlier lowering:

- as you stated, there is and will be a demand to run normal mode dungeons not once but some multiple times to get the required reputation to revered. this will affect everyone and hence creates a demand for dungeons runs.
- practically, there may even be a way to track how often each final dungeon boss has been killed / how many rdfs have been completed to monitor this better
- Attunements were an essential part of classic and tbc content (and practically removed in wotlk). Some attunements (kara) require normal modes only, but some of the attunements for our later raids (t5 and t6) require completion of dungeons in heroic. Latest at that point, when people want to raid with their chars, it will create another wave of people that will "progress" through the dungeon- and reputation-farm process. this will create a larger demand for normal modes and enable more groups to form, too. In the same way, you can start debating if it makes sense to lower this requirements earlier (e.g. mid of t5 content) or as it was in retail (with release of sunwell iirc).
- even earlier, there will be demand to run several heroic dungeons to enable access to Karazhans Nightbane. People who want to complete that chain for access, will have to reach revered for many factions and hence create another set of players that could queue for normal modes if they dont have the reputation yet.

**Comment by Jbubby** [comment]
the more people playing and gearing their alts the better. simple as! Plus I'm pretty sure most long-time players have a fair amount of alts they'd like to gear.

**Comment by shinzo69** [comment]
The current situation just doesn't make fun. I was Enhancement Shaman before and switched to Resto to get faster queues. Did I get faster queues by playing a healer? No. Im trying to get my Lower City/Keepers of Time Rep up to unlock Heroics but it's just impossible. Not to mention that I was in dungeon queue for several hours a few days ago to find Arcatraz Normal. 

Please lower the requirements for Heroic Keys now and not when the first Raid gets released..

OR

Add some Bonus for Tank/Heal to queue. Most Tanks are just q'ing with premades/guildies as they dont want to bother with pugs.

**Comment by Amrx2** [comment]
Covered here - keys are dropping to honored on 12th July. https://www.chromiecraft.com/en/mythic-dungeons-raids-release-schedule/
